openclimatefix/PVConsumer
# Fix the tests and remove docker-compose

* Use `testcontainers` to deal with the database
* Remove all references to sqlite (some of the schema being postgresql-specific)
* Pin some of the dependencies
* Add some dev dependencies
* Mark the tests requiring a live external API as such to make it easy to skip those when running locally
* The tests marked `liveapi` are still broken! See #54
